.travis/script.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e -x
+
+perl Build.PL
```

Comment:
Since we are testing this package, its whole build should be here.
I suggest running 'perl Build.PL' for npg_qc, installing dependencies, then the same for SeqQC, then test both.